source/operators/transcribe/start_transcribe.py

```diff
@@ -44,7 +44,7 @@ def lambda_handler(event, context):
     except KeyError:
         bucket = operator_object.input["Media"]["Video"]["S3Bucket"]
         key = operator_object.input["Media"]["Video"]["S3Key"]
-        file_type = os.path.splitext(key)[1]
+        file_type = key.split('.')[-1]
```

Comment:
This will provide an incorrect file type for files that do not contain a '.' (i.e. don't have an extension). It's best practice to use os.path.splitext because it does not have this problem.

Reply:
`os.path.splitext(key)[1]` didn't actually work, which is why I changed it.